Fix build of swedish help after 33915
Patch by larkang@gmail.com
https://bugs.gentoo.org/show_bug.cgi?id=395677
Applied as r34453 upstream.

Index: help/help_mp-sv.h
===================================================================
--- help/help_mp-sv.h	(revision 34452)
+++ help/help_mp-sv.h	(working copy)
@@ -99,7 +99,6 @@
 #define MSGTR_Playing "Spelar %s.\n"
 #define MSGTR_NoSound "Audio: inget ljud\n"
 #define MSGTR_FPSforced "FPS forcerad att vara %5.3f  (ftime: %5.3f).\n"
-    "För att få bäst prestanda, omkompilera med '--disable-runtime-cpudetection'.\n"
 #define MSGTR_AvailableVideoOutputDrivers "Tillgängliga video-ut-drivrutiner:\n"
 #define MSGTR_AvailableAudioOutputDrivers "Tillgängliga audio-ut-drivrutiner:\n"
 #define MSGTR_AvailableAudioCodecs "Tillgängliga audiocodec:\n"
